cmake/packaging.cmake

```diff
@@ -54,40 +54,33 @@ endif()
 
 set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
 
+# identify proper curl packages for debian
```

Comment:
Is "debian" referring to the distro or the package format (which would include Ubuntu)?

Reply:
package format